rflx/pyrflx/typevalue.py

```diff
@@ -703,6 +724,25 @@ def set_refinement(fld: MessageValue.Field, fld_name: str) -> None:
                     ):
                         fld.typeval.set_refinement(ref.sdu, self._refinements)
 
+        def check_outgoing_condition_satisfied() -> None:
+            if all(
+                [
+                    self.__simplified(o.condition) == FALSE
+                    for o in self._type.outgoing(Field(field_name))
+                ]
+            ):
+                self._fields[field_name].typeval.clear()
+                if isinstance(value, bytes):
+                    value_repr = "x" + value.hex()
```

Comment:
This code is not covered by any test.

Reply:
I think it would be somewhat redundant to test this, because it just changes the formatting of the value representation for byte objects to make the error message on line 740 better readable. The functionality is being tested in `test_ethernet_invalid`. 
Also, i could not think of a way to violate an outgoing edge condition with setting a bytes-value, which is not a too long/too short byte-string, because the correct length is asserted before this line is reached.

Reply:
I'm not sure what you mean with your last sentence. But in general: If a code is reachable, it should be tested. If it is not reachable, it should be removed. We are aiming at 100% test coverage (#334) and this is the single remaining uncovered code in PyRFLX.

Reply:
removed this line as discussed